quickjs.c

```diff
@@ -409,6 +409,12 @@ struct JSContext {
     JSValue global_obj; /* global object */
     JSValue global_var_obj; /* contains the global let/const definitions */
 
+    JSValue regexp_left_ctx; // RegExp.leftContext
+    JSValue regexp_right_ctx; // RegExp.rightContext
+    JSValue regexp_last_match_str;
```

Comment:
You need to mark these in JS_MarkContext, otherwise the GC might free them prematurely.